patches/chromium/.patches

```diff
@@ -137,6 +137,5 @@ backport_1122684.patch
 backport_1111737.patch
 cherry-pick-0e61c69ebd47.patch
 cherry-pick-814a27f8522b.patch
-cherry-pick-adc731d678c4.patch
```

Comment:
Why are these patches removed?

Reply:
Great catch, @zcbenz , this was probably the result of an interactive rebase of the patch commit log in the Chromium source gone wrong. Fixed.